### ISSUE TITLE
Patch apps for update

### ISSUE THREAD

**Issue by coffseducation** [AppImageCommunity/AppImageUpdate#222]
Appimage update has never worked with any apps I have tried. If it does not fail then there is either no option on the icon or it says that it is not an appimage.
I understand the function needs to be enabled by the dev within the app -yes?

Question: Is there a program that can take the appimage and patch in the upgrade function? Preferably sefl updating like the Telegram appimage.

Thanks.

**Comment by probonopd** [comment]
Application authors need to put update information into the AppImage when making it, and need to upload the zsync file alongside the AppImage for it to work.

**Comment by TheAssassin** [comment]
To add some more context, the `zsync` file mentioned by @probonopd is what allows AppImageUpdate to update. This file must be provided by somebody (typically the application's authors). Also, AppImageUpdate requires update information (i.e., it needs to be told where to find said `zsync` file). This information is typically contained within the AppImage. This is recommended for any AppImage. However, not every AppImage's authors take care of this.

TL;DR: If you find an AppImage does not support AppImageUpdate, please ask the application's authors to support AppImageUpdate. We provide all necessary information in our documentation.
